addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -1826,7 +1845,11 @@ ListRenderer.include({
             return;
         }
 
-        this.unselectRow();
+        this.unselectRow({
```

Comment:
i thought this would be the only place where the option should be set.

Reply:
I thought too but if we leave the view or change the page or clicking on a row while there is a new unchanged row then it throws an error